Jantimizer/QueryEstimator/Models/EstimatorResult.cs

```diff
@@ -12,18 +13,24 @@ public class EstimatorResult : ICloneable
         public JsonQuery FromQuery { get; set; }
         public int DidXSweeps { get; set; }
         public ulong EstimatedCardinality { get; set; }
+        public Dictionary<int, List<IBoundResult>> TableAttributeBounds { get; }
+        public Dictionary<int, List<IBoundResult>> FilterBounds { get; }
+        public ISegmentResult? ResultChain { get; }
 
-        public EstimatorResult(JsonQuery fromQuery, ulong estimatedCardinality, int didXSweeps)
+        public EstimatorResult(JsonQuery fromQuery, int didXSweeps, ulong estimatedCardinality, Dictionary<int, List<IBoundResult>> tableAttributeBounds, Dictionary<int, List<IBoundResult>> filterBounds, ISegmentResult? resultChain)
         {
             FromQuery = fromQuery;
-            EstimatedCardinality = estimatedCardinality;
             DidXSweeps = didXSweeps;
+            EstimatedCardinality = estimatedCardinality;
+            TableAttributeBounds = tableAttributeBounds;
+            FilterBounds = filterBounds;
+            ResultChain = resultChain;
         }
 
         public object Clone()
         {
             if (FromQuery.Clone() is JsonQuery query)
-                return new EstimatorResult(query, EstimatedCardinality, DidXSweeps);
+                return new EstimatorResult(query, DidXSweeps, EstimatedCardinality, TableAttributeBounds, FilterBounds, ResultChain);
```

Comment:
`DidXSweeps` sounds like a boolean. Maybe rename to `SweepsDone` or `SweepsDoneCount` ?